scout/server/blueprints/variants/forms.py

```diff
@@ -72,20 +79,20 @@ def process_formdata(self, valuelist):
 class VariantFiltersForm(FlaskForm):
     variant_type = HiddenField(default="clinical")
 
-    gene_panels = SelectMultipleField(choices=[])
+    gene_panels = NonValidatingSelectMultipleField(choices=[])
     hgnc_symbols = TagListField("HGNC Symbols/Ids (case sensitive)")
 
     region_annotations = SelectMultipleField(choices=REGION_ANNOTATIONS)
     functional_annotations = SelectMultipleField(choices=FUNC_ANNOTATIONS)
     genetic_models = SelectMultipleField(choices=GENETIC_MODELS)
 
-    cadd_score = BetterDecimalField("CADD", places=2)
+    cadd_score = BetterDecimalField("CADD", places=2, validators=[validators.Optional()])
     cadd_inclusive = BooleanField("CADD inclusive")
-    clinsig = SelectMultipleField("CLINSIG", choices=CLINSIG_OPTIONS)
+    clinsig = NonValidatingSelectMultipleField("CLINSIG", choices=CLINSIG_OPTIONS)
 
-    gnomad_frequency = BetterDecimalField("gnomadAF", places=2)
+    gnomad_frequency = BetterDecimalField("gnomadAF", places=2, validators=[validators.Optional()])
 
-    filters = SelectField(choices=[])
+    filters = NonValidatingSelectMultipleField(choices=[], validators=[validators.Optional()])
```

Comment:
Yes, i had to do it otherwise the whole form validation fails. We can change the class of the filter in a second moment of we need to, but it's a select, and its content should be already controlled, so I think it's fine to have it like this.

Reply:
Hm, I'm not convinced. The Select becomes a MultiSelect, allowing several filters to be checked. What happens then on load or delete?

Reply:
> Hm, I'm not convinced. The Select becomes a MultiSelect, allowing several filters to be checked. What happens then on load or delete?

It crashes! I'm don't remember much of how the filters select works. Good that you checked!

Reply:
I've changed the class of filters to a non-validating single select class and loading or removing filters now works!